workers/unity/Packages/io.improbable.gdk.testutils/Editor/HeadlessBuildModifier.cs

```diff
@@ -0,0 +1,23 @@
+using Improbable.Gdk.TestUtils.Editor;
+using UnityEditor;
+using UnityEditor.TestTools;
+
+[assembly: TestPlayerBuildModifier(typeof(HeadlessBuildModifier))]
```

Comment:
Does this file belong in this package? As it stands this gets shipped to users and is then applied to their test players builds.

Reply:
Yeah this should go inside `Tests/Performance/PlaymodePerformanceTests`